kfdefs/overlays/moc/smaug/opf-datacatalog/bucket.yaml

```diff
@@ -0,0 +1,11 @@
+---
+apiVersion: objectbucket.io/v1alpha1
```

Comment:
This resource doesn't make sense without NooBaa present on the cluster (see https://github.com/operate-first/apps/pull/1012#pullrequestreview-757312754) I think we don't want it here for now.